api/src/opentrons/protocol_api/labware.py

```diff
@@ -204,10 +204,17 @@ def from_center_cartesian(self, x: float, y: float, z: float) -> Point:
         Specifies a :py:class:`~opentrons.types.Point` based on fractions of the
         distance from the center of the well to the edge along each axis.
 
-        For example, ``from_center_cartesian(0, 0, 0.5)`` specifies a point with no
-        change in the x- or y-axis, and half of the distance from the center of
-        the well to its top along the z-axis. Use :py:meth:`.move` to construct a
-        :py:class:`~opentrons.types.Location` that a pipette can move to.
+        For example, ``from_center_cartesian(0, 0, 0.5)`` specifies a point at the
+        well's center on the x- and y-axis, and half of the distance from the center of
+        the well to its top along the z-axis. To move the pipette to that location,
+        construct a :py:class:`~opentrons.types.Location` relative to the same well::
+
+            location = Location(
+                plate["A1"], plate["A1"].from_center_cartesian(0, 0, 0.5)
+            )
```

Comment:
Oops, I had the argument order swapped in my example. My bad.

```suggestion
            location = Location(
                plate["A1"].from_center_cartesian(0, 0, 0.5), plate["A1"]
            )
```

Reply:
It also has to be `types.Location`. Fixing now.